tests/e2e/tests/critical-path/workbench/command-results.e2e.ts

```diff
@@ -61,37 +60,101 @@ test('Verify that user can see the results found in the table view by default fo
-    await t.switchToIframe(workbenchPage.iframe);
-    await t.expect(await workbenchPage.queryTableResult.visible).ok('The table result is not displayed for command FT.SEARCH');
-    // Select Text view and check result
-    await t.switchToMainWindow();
-    await workbenchPage.selectViewTypeText();
-    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssQueryTextResult).visible).ok('The result is not displayed in Text view');
-});
-// Skipped due to issue https://redislabs.atlassian.net/browse/RI-3524
-test.skip('Verify that user can switches between Table and Text for Client List and see results corresponding to their views', async t => {
+        for (const command of commands) {
+            await workbenchPage.sendCommandInWorkbench(command);
+        }
+        await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssTableViewTypeOption).visible)
+            .ok('The table view is not selected by default for command FT.SEARCH');
+        await t.switchToIframe(workbenchPage.iframe);
+        await t.expect(await workbenchPage.queryTableResult.visible).ok('The table result is not displayed for command FT.SEARCH');
+        // Select Text view and check result
+        await t.switchToMainWindow();
+        await workbenchPage.selectViewTypeText();
+        await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssQueryTextResult).visible).ok('The result is not displayed in Text view');
+    });
+
+test('Verify that user can switches between Table and Text for Client List and see results corresponding to their views', async t => {
     const command = 'CLIENT LIST';
     // Send command and check table view is default
     await workbenchPage.sendCommandInWorkbench(command);
+    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssClientListViewTypeOption).visible)
+        .ok('The table view is not selected by default for command CLIENT LIST');
     await t.switchToIframe(workbenchPage.iframe);
-    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssCustomPluginTableResult).visible).ok('The search results are not displayed in  Custom Table view by default');
+
+    await t.expect(await workbenchPage.queryTableResult.visible)
+        .ok('The search results are not displayed in Client List Table view by default');
     // Select Text view from dropdown and check search results
     await t.switchToMainWindow();
     await workbenchPage.selectViewTypeText();
-    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssQueryTextResult).visible).ok('The result is not displayed in Text view');
+    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssQueryTextResult).visible)
+        .ok('The result is not displayed in Text view');
 });
+
+test.only
```

Comment:
Is that on purpose? better to add in a comments the reason fo

Reply:
I just added it to check that the plugin test will pass on the CircleCI,
removed the attribute now
thanks